### ISSUE TITLE
Update alert/alert-group with a11y changes from core

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#1286]
From this PR - https://github.com/patternfly/patternfly-next/pull/1257/files

* We're no longer using "notification" language in alerts, it's all just "alert" now. I would just search for the word notification and change it to alert in this component.
* The `aria-label` for the close button has been updated to be more descriptive. You can look on core for examples. Here's the docs, you can also find listed on the component page. 

> `aria-label="Close Success alert: Success alert title"` | `.pf-c-button.pf-m-plain` | Indicates the close button. Please provide descriptive text to ensure assistive technologies clearly state which alert is being closed.

* The `.pf-u-screen-reader` screen reader text should also say "Alert" at the end, so instead of "Info:" it would be "Info alert:" (as an example)
    * just an aside, any instance of `.pf-u-screen-reader` should be `.pf-screen-reader` now. You shouldn't have to use any utility class in react now.
* There is a random `>` in the danger alert examples in the workspace
